src/expression.rs

```diff
@@ -98,13 +99,44 @@ impl Index<usize> for SetDirective {
     }
 }
 
+/// A ConvDirective is a command of the form "to hex|dec|bin|oct|dec".
+#[derive(Debug, PartialEq, Eq)]
+pub struct ConvDirective {
+    expr: Expr,
+    radix: String
```

Comment:
This type is better suited to be `FormatRadix`.

Reply:
Ok.